src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/GroupManager.java

```diff
@@ -557,6 +557,16 @@ public static boolean isOwnerOrAdditionalManager(final UserGroupDTO group, final
         return group.getOwnerId().equals(userIdToCheck) || isInAdditionalManagerList(group, userIdToCheck);
     }
 
+    /**
+     * Helper function to check if a user has additional permissions to modify and manage a group.
+     * @param group - dto
+     * @param userIdToCheck - user id to verify
+     * @return whether the user is an owner or an additional manager with privileges.
+     */
+    public static boolean hasAdditionalManagerPrivileges(final UserGroupDTO group, final Long userIdToCheck) {
+        return group.getOwnerId().equals(userIdToCheck) || isInAdditionalManagerList(group, userIdToCheck) && group.isAdditionalManagerPrivileges();
```

Comment:
I'd always put some brackets in since they're free and would make the two conditions clear without relying on knowledge of operator precedence.